BlueprintUI/Tests/EnvironmentTests.swift

```diff
@@ -132,13 +137,26 @@ class EnvironmentTests: XCTestCase {
         let viewDescription = leafViewDescription(
             in: NativeViewNode(
                 content: UIView.describe { _ in },
+                environment: .empty,
                 layoutAttributes: LayoutAttributes(frame: .zero),
                 children: layoutResultNode.resolve()))
 
         viewDescription.apply(to: view)
 
         XCTAssertEqual(view.testValue, testValue)
     }
+    
+    func test_merged() {
+        var lhs = Environment.empty
+        lhs[TestKey.self] = .wrong
+        
+        var rhs = Environment.empty
+        rhs[TestKey.self] = .right
+        
+        let output = lhs.merged(prioritizing: rhs)
+        
+        XCTAssertEqual(output[TestKey.self], .right)
```

Comment:
Would it also be worth testing that setting values for properties exclusively on each side are merged correctly? E.g. only set `KeyA` on `lhs` and `KeyB` on `rhs` and validate that both values exist on the merged object.

Reply:
Sure

Reply:
Done